frontend/iframe.html

```diff
@@ -127,6 +126,5 @@
 
       window.parent.postMessage(JSON.stringify({ inited: true }), '*');
     </script>
-    <script type="text/javascript" src="remark.js"></script>
```

Comment:
Links to files were removed because of webpack-html-plugin inject them automatically.

Reply:
Would you be so kind as to move unrelated changes to separate PR so this one would be crystal-clear? From your commentary, I'm under impression that this change is unrelated to this PR title.

Reply:
I think we won't use this changes.
Also, this changes needs to this PR, I removed static link for files, because webpack generates links with hashes and link it automatically.